src/cosmos/engine/CosmosEngine.ts

```diff
@@ -88,13 +79,12 @@ import {
 import {
   assetFromString,
   checkAndValidateADR36AminoSignDoc,
-  createCosmosClients,
-  coerceToBytes,
-  getIbcChannelAndPort,
+  createCosmosClients, getIbcChannelAndPort,
```

Comment:
There are formatting errors in this commit. You'll need to do a `yarn lint --fix` before rebasing & merging.